<launch>

  <arg name="input_raw_control_cmd" default="/ackermann_cmd_raw"/>
  <arg name="output_filtered_control_cmd" default="/ackermann_cmd"/>
  <arg name="control_filter_param"  default="$(find-pkg-share control_filter)/config/control_filter.param.yaml"/>
  <!-- control filter node -->
  <node pkg="control_filter" 
        exec="control_filter_node"
        name="control_filter_node"
        output="screen">

    <!-- パラメータ -->
    <param from="$(var control_filter_param)"/>
    <!-- トピックのremap -->
    <remap from="/cmd_drive" to="$(var input_raw_control_cmd)"/>
    <remap from="/cmd_drive_filtered" to="$(var output_filtered_control_cmd)"/>

  </node>


</launch>